src/tox_gh_actions/plugin.py

```diff
@@ -39,10 +39,21 @@ def tox_configure(config):
     gh_actions_config = parse_config(config._cfg.sections)
     verbosity2("tox-gh-actions config: {}".format(gh_actions_config))
 
+    if gh_actions_config["envs_are_optional"] is None:
+        warning(
+            "Config 'gh-actions.envs_are_optional' will become the default in a "
+            "future release. Set explicitly to 'true' or 'false' to disable this "
+            "warning."
+        )
```

Comment:
@ymyzk this can be removed if the default won't be changed, and this option probably needs to be added to the documentation.